packages/utils/src/stacktrace.ts

```diff
@@ -115,79 +126,77 @@ function node(getModule?: GetModuleFn): StackLineParserFn {
 
   // eslint-disable-next-line complexity
   return (line: string) => {
-    if (line.match(FILENAME_MATCH)) {
-      return {
-        filename: line,
-      };
-    }
-
     const lineMatch = line.match(FULL_MATCH);
```

Comment:
@AbhiPrasad @timfish do we have a sense for which one of these is more likely to match? We could reorder them and prioritize

Reply:
@timfish wdyt about moving this regexp match after the function match then? It sounds like we would expect that to hit most of the time so we dont waste time on the filename matching - I'm not too familiar with why that file match was there in the first place I'd want to avoid breaking some edge case that we may have just never tested.

Reply:
This code was originally forked from node `stack-trace`. It appears we either lost the comments detailing this fork with the refactoring or the fork comments were never added in the first place. I'll look back through the history to find out. The code bares little resemblance to that original code but we should probably still have those comments since it was the original source.

This also holds true for the [browser code](https://github.com/getsentry/sentry-javascript/blob/develop/packages/browser/src/stack-parsers.ts) which many moons ago came from tracekit.

The original `stack-trace` repository details [the reason for the `------` lines](https://github.com/felixge/node-stack-trace#long-stack-traces) in the readme. It looks like we can safely ignore these stack lines since they're not important to the frames sent to Sentry. Since this doesn't match `FULL_MATCH`, these lines will be dropped.